e2e/debugTarget.js

```diff
@@ -0,0 +1,14 @@
+require('supports-color'); // for prettier logs
+const debug = require('debug');
+const util = require('util');
+const fs = require('fs');
+
+const stream = process.env.DEBUG_TARGET
+  ? fs.createWriteStream(require('path').resolve(process.env.DEBUG_TARGET), { flags: 'a' })
```

Comment:
If ENV set to the file all the debug logs go there, so we can have some more clean test logs + test failure logs in CI